BSP/STM32F413/Src/BSP_SPI.C

```diff
@@ -0,0 +1,162 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "BSP_SPI.h"
+#include "stm32f4xx.h"
+#include "os.h"
+
+
+#define SPI_PORT SPI3
+
+OS_SEM SPI_Update_Sem4;
+
+static void spi_post(void) {
+	OS_ERR err;
+	OSSemPost(&SPI_Update_Sem4, OS_OPT_POST_1, &err);
+	// TODO: error handling
```

Comment:
Ideally, this should be addressed before merging this. If you want to wait on this, at least make a plan or issue to address this.